kedro/extras/datasets/tensorflow/README.md

```diff
@@ -0,0 +1,29 @@
+# TensorFlowModelDataset
+
+``TensorflowModelDataset`` loads and saves TensorFlow models.
+The underlying functionality is supported by, and passes input arguments to TensorFlow 2.X load_model and save_model methods. Only TF2 is currently supported for saving and loading, V1 requires HDF5 and serializes differently.
+
+#### Example use:
+```python
+import numpy as np
+import tensorflow as tf
+
+from kedro.extras.datasets.tensorflow import TensorFlowModelDataset
+
+data_set = TensorFlowModelDataset("tf_model_dirname")
+model = tf.keras.Model()
+predictions = model.predict([...])
+data_set.save(model)
+loaded_model = data_set.load()
+new_predictions = loaded_model.predict([...])
+np.testing.assert_allclose(predictions, new_predictions, rtol=1e-6, atol=1e-6)
```

Comment:
i would put more blank lines in between the different portions (data / save / load / data) but not a big deal